resources/icons/16px/page-break.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect y="7" width="4" height="2" rx="1" fill="#5F6B7C"/>
+<rect x="6" y="7" width="4" height="2" rx="1" fill="#5F6B7C"/>
+<rect x="12" y="7" width="4" height="2" rx="1" fill="#5F6B7C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14 0H12V3H4V0H2V4C2 4.55 2.45 5 3 5H13C13.55 5 14 4.55 14 4V0Z" fill="#5F6B7C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 16H4V13L12 13V16H14V12C14 11.45 13.55 11 13 11L3 11C2.45 11 2 11.45 2 12V16Z" fill="#5F6B7C"/>
+</svg>
```

Comment:
Can we remove the `width`, `height`, `fill`, `fill-rule`, and `clip-rule` attributes from these? They aren't needed.
```svg
<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
<rect y="7" width="4" height="2" rx="1"/>
<rect x="6" y="7" width="4" height="2" rx="1"/>
<rect x="12" y="7" width="4" height="2" rx="1"/>
<path d="M14 0H12V3H4V0H2V4C2 4.55 2.45 5 3 5H13C13.55 5 14 4.55 14 4V0Z"/>
<path d="M2 16H4V13L12 13V16H14V12C14 11.45 13.55 11 13 11L3 11C2.45 11 2 11.45 2 12V16Z"/>
</svg>

```

Reply:
Updated as per suggestion